webplatform/app/views/subscribe/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="row">
```

Comment:
Can you use slim here instead or erb? This way we can write only `.row` intead of `<div class="row"></div>` (so 1 line less ;)) Same with `if` => `<%if%> <%end%>` will be changed to `- if` only.

All fields and submit button will remain the same - just remove opening and closing `<%%>`